src/actions/AaveWelcomeActions.tsx

```diff
@@ -0,0 +1,54 @@
+import { Disklet } from 'disklet'
+import * as React from 'react'
+import FastImage from 'react-native-fast-image'
+import { cacheStyles } from 'react-native-patina'
+import { sprintf } from 'sprintf-js'
+
+import { Space } from '../components/layout/Space'
+import { ButtonsModal } from '../components/modals/ButtonsModal'
+import { Airship } from '../components/services/AirshipInstance'
+import { EdgeText } from '../components/themed/EdgeText'
+import { AAVE_WELCOME } from '../constants/constantSettings'
+import s from '../locales/strings'
+import { borrowPlugins } from '../plugins/helpers/borrowPluginHelpers'
+import { config } from '../theme/appConfig'
+import { Theme } from '../types/Theme'
+import { getBorrowPluginIconUri } from '../util/CdnUris'
+
+export const showAaveWelcomeModal = async (disklet: Disklet, theme: Theme) => {
```

Comment:
Our coding convention is to make the file the modal component which is launched by the scene. Move the logic for when the modal is launched into the scene within the useEffect. Then launch the modal via Airship.show() inside. This file should be called LoanWelcomeModal.tsx much like our other modals. This way the LoanWelcomeModal is an actual component which can access useTheme() on it's own and not needing it passed in.